Update fr.yml for covidshield.app

emilybulger opened the pull request:
Create French version of covidshield.app.

Some content was not translated from original file, so calling in @annemarieleger and @Guillaumegranger1 to translate final sections and add any additional edits/updates to this PR.

emilybulger committed:
Update fr.yml for covidshield.app

emilybulger commented:
@dfmcphee can you also make sure a French language screenshot gets added to this page? If there's an easier way to add per-language screenshots so I don't need to ping you each time, let me know!

annemarieleger commented:
@Guillaumegranger1 
I added the missing translations [here](https://docs.google.com/spreadsheets/d/1qP1RlDTm_xduFxcvN0coobIEZjbkQSjGypruxrEU29c/edit?ts=5ebd4405#gid=557207353) if you could please review the ones with the red checkboxes, thanks!

dfmcphee commented:
> @dfmcphee can you also make sure a French language screenshot gets added to this page? If there's an easier way to add per-language screenshots so I don't need to ping you each time, let me know!

I actually ended up converting the screen shot to HTML and CSS so that we could more easily translate it. These [translation strings](https://github.com/CovidShield/covidshield.app-src/pull/5/files#diff-744657b199290317a1e2e050ad300431R66-R71) are what are rendered in the example notification now.

Guillaumegranger1 commented:
> I added the missing translations here if you could please review the ones with the red checkboxes, thanks!

@annemarieleger I added my recommendations in the doc

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
-    return to work."
+  other: "COVID Shield est une solution de notification d'exposition libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité."
 disclaimer_title:
   other: "How can I download and use COVID Shield?"
```
Missing translation:
`Comment puis-je télécharger et utiliser COVID Shield?`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
 disclaimer_title:
   other: "How can I download and use COVID Shield?"
 disclaimer_message:
   other: "COVID Shield is not currently available for download or use."
```
Missing translation:
`COVID Shield n'est pas disponible actuellement pour usage, ni pour téléchargement.`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
 disclaimer_message:
   other: "COVID Shield is not currently available for download or use."
 disclaimer_content:
   other: "COVID Shield is provided as a reference for your local public health authority to build their own app. If you are interested in using COVID Shield, please contact your local government representative and ask what they are doing about exposure notification for COVID-19."
```
Missing translation:
`COVID Shield est fourni à titre de référence pour que votre autorité locale de santé publique puisse bâtir sa propre application mobile. Si vous êtes intéressé(e) à utiliser COVID Shield, veuillez contacter votre représentant local du gouvernement pour savoir comment y sont traitées les notifications d'exposition à la COVID-19.`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+    'Photo de <a href="https://burst.shopify.com/@alpenglowphoto?utm_campaign=photo_credit&amp;utm_content=Browse+Free+HD+Images+of+Line+Of+Frosty+Pines+Separates+Frozen+Lake+From+Mountains&amp;utm_medium=referral&amp;utm_source=credit">Shawna
+    et Damien Richard</a> de <a href="https://burst.shopify.com/lake?utm_campaign=photo_credit&amp;utm_content=Browse+Free+HD+Images+of+Line+Of+Frosty+Pines+Separates+Frozen+Lake+From+Mountains&amp;utm_medium=referral&amp;utm_source=credit">Burst</a>'
 return_to_homepage:
   other: "Return to homepage"
```
Missing translation:
`Retourner à la page d'accueil`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
-    with privacy as its top priority. It was built by a group of
-    volunteers in order to help Canadians and the rest of the world safely
-    return to work."
+  other: "COVID Shield est une solution de notification d'exposition libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité."
```
We want to use "notification d'exposition **à la COVID-19**" as we did in the app so I suggest:
`COVID Shield est une solution de notification d'exposition à la COVID-19 libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité.`

Guillaumegranger1 replied:
notification with a ¨s¨ ?

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
-    href="https://www.apple.com/covid19/contacttracing">provided by Apple
-    and Google</a>, the most privacy-preserving approach currently
-    available.'
+  other: 'COVID Shield est un outil privé, sécurisé et facile à utiliser, qui vise à aider les gouvernements à lancer leurs propres systèmes de notification d'exposition. L'outil est basé sur une technologie de notification d'exposition <a
```
We want to use "notification d'exposition **à la COVID-19**" as we did in the app so I suggest:
`COVID Shield est un outil privé, sécurisé et facile à utiliser, qui vise à aider les gouvernements à lancer leurs propres systèmes de notification d'exposition à la COVID-19. L'outil est basé sur une technologie de notification d'exposition`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
-    is flexible enough to be deployed at various levels of government by
-    enabling the server and mobile app to talk to each other even if there
-    are multiple instances."
+  other: "Pour que l'efficacité du système de notification d'exposition soit optimale, il est important que les gouvernements lancent des applications compatibles les unes avec les autres. COVID Shield est suffisamment flexible pour être déployé à différents ordres de gouvernement en activant le serveur et l'application mobile de manière qu'ils communiquent même en présence de plusieurs instances ...
```
We want to use "notification d'exposition **à la COVID-19**" as we did in the app so I suggest:
`Pour que l'efficacité du système de notification d'exposition à la COVID-19 soit optimale, il est important que les gouvernements lancent des applications compatibles les unes avec les autres. COVID Shield est suffisamment flexible pour être déployé à différents ordres de gouvernement en activant le serveur et l'application mobile de manière qu'ils communiquent même en présence de plusieurs instances.`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "Articles connexes"
 related_reading_1:
-  other: "How does Exposure Notification work?"
+  other: "Comment fonctionne la notification d'exposition ?"
```
We want to use "notification d'exposition à la COVID-19" as we did in the app so I suggest:
`Comment fonctionne la notification d'exposition à la COVID-19 ?`

annemarieleger replied:
Is there a reason why "Exposure Notification" in English is singular and capitalized?

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "Comment fonctionne la notification d'exposition ?"
 related_reading_2:
-  other: "Why is Exposure Notification important?"
+  other: "Pourquoi la notification d'exposition est-elle importante ?"
```
We want to use "notification d'exposition à la COVID-19" as we did in the app so I suggest:
`Pourquoi la notification d'exposition à la COVID-19 est-elle importante ?`

Guillaumegranger1 replied:
@annemarieleger  I suggest we remove the space before the `?`

annemarieleger replied:
No strong opinion. If we want to remove it from here, we would have to remove it everywhere there's a "?"
Would you remove it everywhere there is a ":" as well?

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "Application mobile (iOS et Android)"
 mobile_app_content:
-  other: "The mobile app runs in the background and requires no user interaction after onboarding. It uses Bluetooth to collect and share random IDs with nearby phones with COVID Shield installed. If a user tests positive for COVID-19, they can choose to anonymously share their data so others can be notified of possible exposure. Sharing of random IDs is voluntary and only possible with a positiv ...
+  other: "L'application mobile fonctionne en arrière-plan et n'exige aucune interaction avec l'utilisateur après l'intégration. Elle utilise le Bluetooth pour recueillir et partager des identifiants aléatoires avec les téléphones situés à proximité, sur lesquels COVID Shield est installé. Si un utilisateur est testé positif au COVID-19, il peut choisir de partager anonymement ses données de maniè ...
```
Couple of things in this paragraph. 
**1-** "intégration" doesn't make much sense for an app onboarding (_vs_ an in-person onboarding), I suggest "processus d'accueil"
**2-** "testé positif" is an anglicism, we use "reçoit un résultat de test positif"
**3-** we say "à la" COVID-19, and not "au" COVID-19
So this paragraph becomes:
`L'application mobile fonctionne en arrière-plan et n'exige aucune interaction avec l'utilisateur après le processus d'accueil. Elle utilise le Bluetooth pour recueillir et partager des identifiants aléatoires avec les téléphones situés à proximité, sur lesquels COVID Shield est installé. Si un utilisateur reçoit un résultat de test positif à la COVID-19, il peut choisir de partager anonymement ses données de manière que d'autres puissent être avisés d'une possible exposition. Le partage des identifiants aléatoires se fait sur une base volontaire et n'est possible que si le résultat positif du test a été confirmé par un professionnel de santé. COVID Shield télécharge périodiquement les identifiants aléatoires partagés depuis le serveur et les compare sur l'appareil de chaque utilisateur pour déterminer si une exposition possible a eu lieu.`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "<strong>L'approche de la notification d'exposition adoptée par COVID Shield place la confidentialité au rang de priorité.</strong> L'application COVID Shield, son serveur et le portail web de résultats sont tous trois conçus pour protéger la vie privée tout en aidant les gens à reprendre le travail en toute sécurité."
 privacy_action:
-  other: "Read our sample privacy policy"
+  other: "Lire notre politique de confidentialité"
```
"Sample" was not translated, so I suggest: 
`Lire notre exemple de politique de confidentialité`

Guillaumegranger1 replied:
@annemarieleger
• Line 31: I suggest we remove the space before the ?

annemarieleger replied:
Ok, will change the "w" for capital "W".

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "Serveur"
 server_content:
-  other: "The server securely collects and stores random IDs uploaded from the mobile app after a confirmed positive test result. It also generates unique temporary codes which grant permission for people with a positive COVID-19 test result to upload their random IDs. The server is flexible enough to be deployed at various levels of government."
+  other: "Le serveur recueille et stocke de façon sécurisée les identifiants aléatoires importés depuis l'application mobile suite à un résultat de test positif confirmé. Il génère également des codes temporaires uniques qui accordent aux individus testés positifs au COVID-19 l'autorisation de mettre en ligne leurs identifiants aléatoires. Le serveur est suffisamment flexible pour un déploiement  ...
```
1- "testé positif" is an anglicism, we use "reçoit un résultat de test positif"
2- we say "à la" COVID-19, and not "au" COVID-19
So this paragraph becomes: 
`Le serveur recueille et stocke de façon sécurisée les identifiants aléatoires importés depuis l'application mobile suite à un résultat de test positif confirmé. Il génère également des codes temporaires uniques qui accordent aux individus ayant reçu un résultat de test positif à la COVID-19 l'autorisation de mettre en ligne leurs identifiants aléatoires. Le serveur est suffisamment flexible pour un déploiement à différents ordres de gouvernement.`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "Portail web"
 web_portal_content_1:
-  other: "The web-based results portal provides health care professionals unique temporary codes which can be shared with users who have tested positive for COVID-19. This code gives a user access to upload their random IDs via the mobile app if they choose. No personal information is collected and there is no association between the codes and specific tests."
+  other: "Le portail web de résultats fournit aux professionnels de santé des codes temporaires uniques qui peuvent être partagés avec les utilisateurs ayant été testés positifs au COVID-19. Ces codes donnent à l'utilisateur un accès lui permettant de mettre en ligne ses identifiants aléatoires via l'application mobile, s'il le souhaite. Aucun renseignement personnel n'est recueilli et il n'exist ...
```
1- "testé positif" is an anglicism, we use "reçoit un résultat de test positif"
2- we say "à la" COVID-19, and not "au" COVID-19
3- we say "professionnels de **la** santé"
So this paragraph becomes:
`Le portail web de résultats fournit aux professionnels de la santé des codes temporaires uniques qui peuvent être partagés avec les utilisateurs ayant reçu un résultat de test positif à la COVID-19. Ces codes donnent à l'utilisateur un accès lui permettant de mettre en ligne ses identifiants aléatoires via l'application mobile, s'il le souhaite. Aucun renseignement personnel n'est recueilli et il n'existe aucune association entre les codes et des tests particuliers.`

Guillaumegranger1 replied:
My suggestions @annemarieleger  : 
• portail Web (with a capital `W`)
• professionnels de `la` santé

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "Qui a conçu COVID Shield ?"
 who_built_content_1:
-  other: "We are a group of Shopify volunteers who want to help to slow the spread of COVID-19 by offering our skills and experience developing scalable, easy to use applications. We are releasing COVID Shield free of charge with a flexible open-source license."
+  other: "Nous sommes un groupe de bénévoles Shopify, et nous souhaitons contribuer au ralentissement de la pandémie de COVID-19 en mettant nos compétences et notre expérience au service du développement d'applications évolutives et conviviales. Nous lançons COVID Shield gratuitement, avec une licence open source flexible."
```
**1-** "bénévoles Shopify" is not clear, I would suggest "bénévoles **de chez** Shopify"
**2-** We decided to use "transmission" to translate "spread" instead of "pandemic"
**3-** we say "de la" COVID-19, and not "de" COVID-19
so this paragraph becomes:
`Nous sommes un groupe de bénévoles de chez Shopify, et nous souhaitons contribuer au ralentissement de la transmission de la COVID-19 en mettant nos compétences et notre expérience au service du développement d'applications évolutives et conviviales. Nous lançons COVID Shield gratuitement, avec une licence open source flexible.`

Guillaumegranger1 replied:
Can (should) `open source` be italicised?  🤔

annemarieleger replied:
I've seen it in the [Larousse](https://www.larousse.fr/dictionnaires/francais/open_source/188163?q=open+source#11058413)? We could translate it by "libre de droit" or "à code ouvert". I guess if the other translator didn't translate it it's because it's intentional to keep an English-tech word?

Guillaumegranger1 replied:
Yes. Hence my suggestion to italicise. Otherwise, we could also put it between french quotes as so: « open source »

annemarieleger committed:
Added some missing translations

Added translations for 
- twitter_title and twitteR_description
- notification_time, notification_title and notification_content

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -1,83 +1,71 @@
+  other: "COVID Shield – une appli de notifications d'exposition à la COVID-19"
 twitter_description:
-  other: "COVID Shield is a free exposure notification solution built with privacy as its top priority."
+  other: "COVID Shield est une solution de notification d'exposition libre d'accès, conçue dans le respect absolu de la confidentialité."
```
I've made a mistake, missing an "s" to "notification". This should read:
`COVID Shield est une solution de notifications d'exposition libre d'accès, conçue dans le respect absolu de la confidentialité.`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -1,83 +1,71 @@
+  other: "La confidentialité est notre priorité absolue"
 privacy_content:
-  other: "<strong>COVID Shield takes a privacy-first approach to exposure notification.</strong> The COVID Shield app, its server, and the web-based results portal are all designed to preserve personal privacy while helping people safely return to work."
+  other: "<strong>L'approche de la notification d'exposition adoptée par COVID Shield place la confidentialité au rang de priorité.</strong> L'application COVID Shield, son serveur et le portail web de résultats sont tous trois conçus pour protéger la vie privée tout en aidant les gens à reprendre le travail en toute sécurité."
```
"web" should have a capital "w", so this should read:
`<strong>L'approche de la notification d'exposition adoptée par COVID Shield place la confidentialité au rang de priorité.</strong> L'application COVID Shield, son serveur et le portail Web de résultats sont tous trois conçus pour protéger la vie privée tout en aidant les gens à reprendre le travail en toute sécurité.`

annemarieleger commented on i18n/fr.yaml:
```diff
@@ -1,83 +1,71 @@
+  other: "Le serveur recueille et stocke de façon sécurisée les identifiants aléatoires importés depuis l'application mobile suite à un résultat de test positif confirmé. Il génère également des codes temporaires uniques qui accordent aux individus testés positifs au COVID-19 l'autorisation de mettre en ligne leurs identifiants aléatoires. Le serveur est suffisamment flexible pour un déploiement  ...
 web_portal_title:
-  other: "Web portal"
+  other: "Portail web"
```
"web" should be with a capital "w", so this should be:
`Portail Web`

Commit:
Updating French copy and config

dfmcphee commented:
I think I have now updated all the translations here based on all of the feedback. I also updated the config so we show the language switcher once this merges. Any other feedback for this?

annemarieleger commented:
Nop, nothing new since my last review!

dfmcphee commented:
> Nop, nothing new since my last review!

Great! Thanks for all of this! 🙏
